test/handlers/redeem_and_unstake/RedeemIntentDeclaredHandler/persist.test.ts

```diff
@@ -0,0 +1,85 @@
+
+import BigNumber from 'bignumber.js';
+import sinon from 'sinon';
+import * as Web3Utils from 'web3-utils';
+
+import RedeemIntentDeclaredHandler from '../../../../src/handlers/redeem_and_unstake/RedeemIntentDeclaredHandler';
+import Message from '../../../../src/models/Message';
+import {
+  MessageDirection, MessageRepository, MessageStatus, MessageType,
+} from '../../../../src/repositories/MessageRepository';
+import assert from '../../../test_utils/assert';
+import SpyAssert from '../../../test_utils/SpyAssert';
+
+describe('RedeemIntentDeclaredHandler.persist()', (): void => {
+  const transactions = [{
```

Comment:
Can we have more than 1 transaction here?

Reply:
With same messageHash right?

Reply:
no different message hash.

Reply:
Done!